clusters/build-clusters/arm01/assets/multi-arch-ci-tools_imagestream.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: multi-arch-ci-tools
+spec:
+  tags:
+  - name: pod-scaler
+    from:
+      kind: DockerImage
+      name: 'quay.io/multi-arch/ci-tools:pod-scaler'
```

Comment:
That might work too, yea. However, we'd like to avoid doing cross-builds - that is exactly what the ARM build cluster is intended for. If this image is not a requirement for scheduling builds on the ARM cluster, it SGTM. Otherwise, I'd like to use this image as a temporary solution until we have builds going on the ARM cluster and committed to the CI registry.

Reply:
It is not required. I believe the only component that really is required in the build clusters is the namespace-ttl-controller and I would ask you to just converge that thing into our central controller manager, so we don't need to have a build for each architecture one might want to have

Reply:
@alvaroaleman in other words, you'd like me to merge the `ci-ns-ttl-controller` into `openshift-controller-manager`?

Reply:
No, into the dptp-controller-manager here: https://github.com/openshift/ci-tools/tree/master/cmd/dptp-controller-manager

Reply:
Unless we want to mess with cross-cluster admission control I think we would be looking at deploying one `pod-scaler` admission controller per cluster?